### PR TITLE
#1134 DHFPROD-1061 - deploy on mapping update now

### PR DESCRIPTION
**Pull request by aebadirad**

### CODE REVIEW

File: quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java [hunk 1]
```diff
@@ -66,10 +69,6 @@ public MappingModel createMapping(String projectDir, MappingModel newMapping) th
         scaffolding.createMappingDir(newMapping.getName());
         Mapping mapping = mappingManager.createMappingFromJSON(newMapping.toJson());
         mappingManager.saveMapping(mapping);
```
Comment (srinathgit) [inline quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java]: Should this piece of code be removed ? After a mapping is created, we do have to install user modules right ? Also deleteMapping() still unwatches the mapping dir when called

File: quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java [hunk 2]
```diff
@@ -66,10 +69,6 @@ public MappingModel createMapping(String projectDir, MappingModel newMapping) th
...
-        Path dir = envConfig().getMlSettings().getHubMappingsDir().resolve(newMapping.getName());
-        if (dir.toFile().exists()) {
-            watcherService.watch(dir.toString());
-        }
         return getMapping(mapping.getName());
     }
```
Comment (srinathgit) [inline quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java]: Should this piece of code be removed ? After a mapping is created, we do have to install user modules right ? Also deleteMapping() still unwatches the mapping dir when called

Comment (aebadirad) [reply quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java]: Yes, it adds another watch that interferes with the original watch laid over the plugin directory, which causes a race condition from time to time.

And as delete mapping removes a directory, so to be safe we force-remove any watch that maybe placed on it as a precautionary measure.

**Comment by ayuwono** [comment]
Tested on DHFPROD 1061 and 1065 combined branch, still seeing the error
```
10:44:23.513 [http-nio-8080-exec-3] ERROR o.a.c.c.C.[.[.[.[dispatcherServlet] - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is com.marklogic.client.ResourceNotFoundException: Local message: Could not read resource at resources/ml:scaffoldContent. Server Message: The requested mapping: MapSku-1 was not found] with root cause
com.marklogic.client.ResourceNotFoundException: Local message: Could not read resource at resources/ml:scaffoldContent. Server Message: The requested mapping: MapSku-1 was not found
	at com.marklogic.client.impl.OkHttpServices.checkStatus(OkHttpServices.java:4279)
	at com.marklogic.client.impl.OkHttpServices.getIteratedResourceImpl(OkHttpServices.java:3080)
	at com.marklogic.client.impl.OkHttpServices.getIteratedResource(OkHttpServices.java:3057)
	at com.marklogic.client.impl.ResourceServicesImpl.get(ResourceServicesImpl.java:61)
	at com.marklogic.client.impl.ResourceServicesImpl.get(ResourceServicesImpl.java:57)
	at com.marklogic.hub.scaffold.impl.ScaffoldingImpl$ContentPlugin.getContents(ScaffoldingImpl.java:521)
	at com.marklogic.hub.scaffold.impl.ScaffoldingImpl.createFlow(ScaffoldingImpl.java:118)
	at com.marklogic.quickstart.service.EntityManagerService.createFlow(EntityManagerService.java:317)
	at com.marklogic.quickstart.web.EntitiesController.createFlow(EntitiesController.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:209)
```

**Pull request closed without merging**